src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -35,30 +47,46 @@ public LineResponse saveLine(LineRequest request) {
     }
 
     public LineResponse getLine(Long lineId) {
-        Line findLine = getLineById(lineRepository, lineId);
+        Line findLine = lineRepository.findLineById(lineId);
         return new LineResponse(findLine);
     }
 
     public void modifyLine(Long lineId, LineRequest lineRequest) {
-        Line findLine = getLineById(lineRepository, lineId);
-        findLine.chageLine(lineRequest.getName(), lineRequest.getColor());
+        Line findLine = lineRepository.findLineById(lineId);
+        findLine.changeLine(lineRequest.getName(), lineRequest.getColor());
```

Comment:
```suggestion
        findLine.change(lineRequest.getName(), lineRequest.getColor());
```
Line 객체의 행동을 메서드로 표현한 것이니 line을 또 표현해줄 필요는 없겠네요